### ISSUE TITLE
Template 10 1.1.1.12 + Newtonsoft.Json 11+  causes error

### ISSUE THREAD

**Issue by Meister1867** [Windows-XAML/Template10#1646]
After upgrading my packages, I received an error.  I was able to isolate the issue to moving from version 10+ of Newtonsoft.Json 10+ to Newtonsoft.Json 11+.  The error generated was. 

```
Exception thrown: 'System.MissingMethodException' in Template10Library.dll
An exception of type 'System.MissingMethodException' occurred in Template10Library.dll but was not handled in user code
Method not found: 'Void Newtonsoft.Json.JsonSerializerSettings.set_TypeNameAssemblyFormat(System.Runtime.Serialization.Formatters.FormatterAssemblyStyle)'.
```

**Comment by JerryNixon** [comment]
I hate this recurring error. It has to do with the definition of the dependency in nuget's NUPKG syntax, I think. You are not the first to have this experience. The easiest solution is to downgrade your version of NewtonSoft for now. I will leave this as a backlog item to fix in future iterations.

**Comment by Quico20** [comment]
In your template 10.1 V1.1.13b it seems you were solving this issue, I was able to get rid of that error by using your source code and upgrading the libraries.
Now I'm using T10.1 V1.1.13b Custom to my need my only problem is that designer get broken it shows an exception casting from DelegateCommand.

**Comment by moongaze** [comment]
According to [this](https://github.com/JamesNK/Newtonsoft.Json/issues/1671) "TypeNameAssemblyFormat" is deprecated:

"TypeNameAssemblyFormat is obsolete because it could cause this error on some platforms. You should use TypeNameAssemblyFormatHandling instead."

**Comment by FlorisDevreese** [comment]
Interestingly I only had this problem when I updated the Target min version of the project to Windows 10 Fall Creators Update (or newer). 

I fixed the issue by updating the Newtonsoft.Json Nuget dependency to 10.0 3 instead of 11.x.x.
After doing that I had no problem.

Thanks for pointing out it was the combination of template 10 and Newtonsoft.Json 11 that caused the issue. I could have lost ages searching on this one!
